docs/scripts/fledge_plugin_list

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+function table {
+	list="$1"
+	title=$2
+	tableType=$3
+	echo ".. list-table:: $title" >> $output
+	echo "    :widths: 20 50" >> $output
+	echo "    :header-rows: 1" >> $output
+	echo "    :stub-columns: 1" >> $output
+	echo "" >> $output
+	echo "    * - Name" >> $output
+	echo "      - Description" >> $output
+	for repo in $list
+	do
+		product=`echo $repo | sed -e 's/-.*//'`
+		if [ $product = "foglamp" ]; then
+			org=dianomic
+		else
+			org=fledge-iot
+		fi
+		type=`echo $repo | sed -e 's/fledge-//' -e 's/foglamp-//' -e 's/-.*//'`
+		name=`echo $repo | sed -e 's/fledge-//' -e 's/foglamp-//' -e "s/${type}-//"`
+		echo $repo, $name, $type >>/tmp/repos
+		if [ "$type" = "$tableType" ]; then
+			description=`curl -s -H "$header" https://api.github.com/repos/$org/$repo | jq '.description' | sed -e 's/"//g'`
```

Comment:
In case of White-labelled description always returns null, because it has a check of organization and that repo does not exist on dianomic org.

Reply:
This is never run over the white labelled code, only on fledge, so there is no null case.